test/plugins/commands.test.js

```diff
@@ -22,15 +22,15 @@ describe('commands plugin test', () => {
         });
     });
 
-    beforeEach(() => {
+    beforeEach(async () => {
         // eslint-disable-next-line global-require
         plugin = require('../../plugins/commands');
 
         server = Hapi.server({
             cache: {
-                engine: catmemory,
-                maxByteSize: 512,
-                allowMixedContent: true
+                engine: new Catmemory({
+                    maxByteSize: 512
```

Comment:
Is `allowMixedContent: true` implicit now?

Reply:
Yes, its no longer available. 
https://hapi.dev/module/catbox-memory/api/?v=5.0.0#options